build.gradle

```diff
@@ -101,6 +101,7 @@ ext {
     espressoVersion = '3.5.0'
     jsonVersion = '20220924'
     uiautomator = '2.2.0'
+    showkaseVersion = '1.0.0-beta14'
```

Comment:
nit: if this is only used in financial-connections, could we just define this within?